react_week1/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './App.css';
+import RenderTodo from './todo.js'
+
+const toDoList = [
+{
+  description: 'Get out of bed',
+  deadLine: 'Wed Sep 13 2017' 
+},
+{
+  description: 'Brush teeth',
+  deadLine: 'Thu Sep 14 2017' 
+},
+{
+  description: 'Eat breakfast',
+  deadLine: 'Fri Sep 15 2017'
+}
+];
+
+function App() {
+  return (
+    <div className="todo-App">
```

Comment:
it is a little more idiomatic to use "todo-app" here. Attributes like these are generally case insensitive.